vm/systemSmartContracts/governance_test.go

```diff
@@ -2450,6 +3065,67 @@ func TestGovernanceContract_IsValidVoteString(t *testing.T) {
 	require.False(t, gsc.isValidVoteString("invalid"))
 }
 
+func TestComputeEndResults(t *testing.T) {
+	t.Parallel()
+
+	args := createMockGovernanceArgs()
+	args.Eei = &mock.SystemEIStub{
+		GetStorageCalled: func(key []byte) []byte {
+			if bytes.Equal(key, []byte(governanceConfigKey)) {
+				configBytes, _ := args.Marshalizer.Marshal(&GovernanceConfigV2{
+					MinQuorum: big.NewInt(100),
+					MinPassThreshold: big.NewInt(51),
+					MinVetoThreshold: big.NewInt(30),
+				})
+				return configBytes
+			}
+
+			return nil
+		},
+	}
+	gsc, _ := NewGovernanceContract(args)
+
+	didNotPassQuorum := &GeneralProposal{
+		Yes: big.NewInt(50),
+		No: big.NewInt(0),
+		Veto: big.NewInt(0),
+	}
+	gsc.computeEndResults(didNotPassQuorum)
```

Comment:
maybe treat the error here and in the other 4 places from this test function?

Reply:
done